src/index.js

```diff
@@ -94,3 +110,53 @@ function generateTripCosts(traveler) {
   let includingAgentFee = (totalSpent * 1.1);
   return includingAgentFee
 }
+
+function generateNewTrip() {
+
+
+}
+
+function validateDateEntry() {
+  let validDate;
+  let dateInput = document.querySelector('.date-input')
+  if(moment(dateInput.value)._isValid || moment(dateInput.value).isAfter(moment(Date.now()))) {
+    validDate = true
+  } else {
+    validDate = false
+  }
+  return validDate
+}
+
+function validateDuration() {
+  let validNum;
+  let durationInput = document.querySelector('.duration-input');
+  if(typeof durationInput.value == 'number' || durationInput.value > 1) {
+    validNum = true
+  } else {
+    validNum = false
+  }
+  return validNum
+}
+
+function validateTravelers() {
+  let validNum;
+  let travelerInput = document.querySelector('.traveler-input');
+  if(typeof travelerInput.value == 'number' || travelerInput.value > 0) {
+    validNum = true
+  } else {
```

Comment:
I assume this works, but wouldn't it need to be &&? You want something that is a number _and_ greater than 0? Maybe I'm overthinking it.

Reply:
You know, I tried `&&`, because I was thinking the same thing, but it always came back as `false`! Super odd. Works perfectly fine with the `||`. I'm just rolling with it at this point.

Reply:
I think the .value is coming in as a string. Try console.log(typeof(travelerInput.value)). This is probably why your && condition wasn't working. Also you can use isNaN(travelerInput.value) to return a boolean of whether travelerInput.value is "Not a Number" or "Not NOT a Number"

Reply:
oh neat! that'll be good to know for future projects, thank you!